messages/en.json

```
@@ -1,8 +1,11 @@
 {
   "store/list-price.default": "Original price: {listPriceValue}",
   "store/savings.default": "Selling price: {newPriceValue}",
+  "store/installments.default": "or {installmentsNumber}x of {installmentValue} (total = {installmentsTotalValue})",
```

Weird default.

Add the `hasInterestRate` value to render the interest rate or render "interest-free"
```suggestion
  "store/installments.default": "or {installmentsNumber}x of {installmentValue} {hasInterestRate, select, true {",
```